code/ai/aicode.cpp

```diff
@@ -11523,7 +11508,7 @@ void render_wing_phantoms(object *objp)
 
 	for (i=0; i<32; i++)
 		if (Debug_render_wing_phantoms & (1 << i)) {
-			get_absolute_wing_pos(&goal_point, objp, i, 0);
+			get_absolute_wing_pos(&goal_point, objp, shipp->wingnum, i, false);
```

Comment:
Just use plain old `wingnum` instead of `shipp->wingnum` here because that's what's being referenced earlier in the function